ci-automation/capi_image.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# >>> This file is supposed to be SOURCED from the repository ROOT. <<<
```

Comment:
If we don't really need sourcing, we can simplify this and expect the script to be invoked directly. Sourcing always modifies shared global state and is hard to reason about.